### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by stephenkhess**
readStreamTest.js test is a bit tedious but setup wasn't all that bad.  Alternatively the strategy would be to inject either an array of streams or an options object containing all the requested streams.  I'd prefer the former but I couldn't figure out how to iterate a list and chain together the pipes.

### CODE REVIEW

**Comment by stephenkhess** [comment]
Fixed #12

File: test/readStreamTest.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+var readStream = require('../src/readStream');
+
+tape('readStream', function(test) {
+  /*
+    this test is not terribly attractive, i'm not happy with it but setup wasn't
+    all that painful.
+  */
+  test.test('readStream should return from all requested types and populate wofRecords', function(t) {
+    // remove tmp directory if for some reason it's been hanging around from a previous run
+    fs.removeSync('tmp');
```
Comment (dianashk) [inline test/readStreamTest.js]: I would pull all the file writing and dir creating into a setup function so it's more clear where the test actually begins.

Comment (stephenkhess) [inline test/readStreamTest.js]: Good call, will do

**Comment by orangejulius** [comment]
:+1: :+1:

**Pull request merged**